packages/plugins/i18n/server/src/register.ts

```diff
@@ -46,37 +46,35 @@ const addContentTypeSyncHooks = (strapi: Strapi) => {
 };
 
 /**
- * Adds locale and localization fields to localized content types
+ * Adds locale and localization fields to all content types
+ * Even if content type is not localized, it will have these fields
  * @param {Strapi} strapi
  */
-const extendLocalizedContentTypes = (strapi: Strapi) => {
-  const contentTypeService = getService('content-types');
+const extendContentTypes = (strapi: Strapi) => {
   const coreApiService = getService('core-api');
 
   Object.values(strapi.contentTypes).forEach((contentType) => {
-    if (contentTypeService.isLocalizedContentType(contentType)) {
-      const { attributes } = contentType;
+    const { attributes } = contentType;
 
-      _.set(attributes, 'localizations', {
-        writable: true,
-        private: false,
-        configurable: false,
-        visible: false,
-        type: 'relation',
-        relation: 'oneToMany',
-        target: contentType.uid,
-      });
+    _.set(attributes, 'localizations', {
```

Comment:
Aren't we deleting those entirely ?

Reply:
Yes we are going to be removing it in a later PR! 

I am keeping it for now to keep the UI stable and working (so we can select locales) It makes it easier for testing features atm!